lib/screens/history/widgets/edit_bottomsheet.dart

```diff
@@ -54,8 +54,8 @@ class _EditBottomSheetState extends State<EditBottomSheet> {
               shape: RoundedRectangleBorder(
                 borderRadius: BorderRadius.circular(10.toWidth),
               ),
-              content: ConfirmationDialog(TextStrings.deleteFileConfirmationMsg,
-                  () async {
+              content: ConfirmationDialog(
+                  TextStrings.deleteFileConfirmationMsgMyFiles, () async {
                 widget.onConfirmation();
```

Comment:
Can you check if edit bottomsheet is only being called for deleting my files items?
Because `deleteFileConfirmationMsgMyFiles`  will be a fixed text here.

Reply:
Added a new parameter to edit-bottomsheet, which carries a delete message from where it is called.